.scripts/start.js

```diff
@@ -110,10 +110,7 @@ function startApp(callback) {
 function startChimp() {
 	startProcess({
 		name: 'Chimp',
-		command: 'npm',
```

Comment:
*NO PR*

This changes the default test command to utilize our one-by-one-testing mechanism